README.md

```diff
@@ -7,4 +7,74 @@ It can be used in a Logstash pipeline to perform the transformations that are ap
 ## Documentation
 
 Logstash provides infrastructure to automatically generate documentation for this plugin from its [asciidoc source](docs/index.asciidoc).
-All plugin documentation are placed under one [central location](http://www.elastic.co/guide/en/logstash/current/).
\ No newline at end of file
+All plugin documentation are placed under one [central location](http://www.elastic.co/guide/en/logstash/current/).
+
+## Development
+### Prerequisites
+- Clone the [plugin repo](https://github.com/elastic/logstash-filter-elastic_integration)
+- Check out the branch you would like to build, such as `8.x` or `9.x/main`, etc...
+  - Note that, `8.series` branches require JDK17 and `9.series`/`main` requires JDK21.
+- Set the Logstash source path in the `gradle.properties` file. Open the file and update the `LOGSTASH_PATH`.
+- Make sure Logstash is compiled.
+- Point to the Elasticsearch branch/version to build the plugin with. Open the `gradle.properties` file and update `ELASTICSEARCH_TREEISH`.
+- Point to the Elasticsearch branch/version to build the plugin with. Open the `gradle.properties` file and update `ELASTICSEARCH_TREEISH`.
```

Comment:
```suggestion
```
duplicate?

Reply:
Yup, indeed duplicate, thank you for catching.